packages/grid/x-data-grid-premium/src/DataGridPremium/DataGridPremium.tsx

```diff
@@ -461,6 +461,12 @@ DataGridPremiumRaw.propTypes = {
    * @default false
    */
   hideFooterSelectedRowCount: PropTypes.bool,
+  /**
+   * If `true`, the diacritics (accents) are ignored when filtering or quick filtering.
+   * * E.g. when filter value is `cafe`, the rows with `café` will be visible.
+   * @default false
+   */
+  ignoreDiacriticsInFiltering: PropTypes.bool,
```

Comment:
Not sure I like the naming, but I don't have anything better to propose atm.

Reply:
Nit: To me personally, simply `ignoreDiacritics` also makes sense, as the JSDoc comment will be sufficient to explain the relation with filtering. Also, the user using it should already be aware of what it does so maybe a little bit less explanatory title would make the prop name a bit simple?